tests/unit/test-cmd/test.build.js

```diff
@@ -72,6 +73,40 @@ describe('build', () => {
     });
   });
 
+  it('throws on missing manifest properties in filename template', () => {
+    return withTempDir(
+      (tmpDir) =>
+        build({
+          sourceDir: fixturePath('minimal-web-ext'),
+          artifactsDir: tmpDir.path(),
+          filename: '{missingKey}-{version}.xpi',
+        })
+          .then(makeSureItFails())
+          .catch((error) => {
+            log.info(error);
```

Comment:
Nit, these long.info(error) are not really needed, I just noticed that we did already have one which got merge in a old patch, but not a big deal. We will have to rewrite all these tests into async functions at some point (to make them a bit more readable), and so don't worry about that.